dragon-directory.js

```diff
@@ -123,6 +124,23 @@ function extract_students(record) {
   return students;
 }
 
+// Map grade to value that can be compared and rendered
+function normalize_grade(grade) {
```

Comment:
when is this called? I don't see it used...

Reply:
lol. I forgot to call it - made another commit that includes function call.